src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/ReduxProvider/TeiWorkingListsReduxProvider.container.js

```diff
@@ -12,31 +14,43 @@ export const TeiWorkingListsReduxProvider = ({ storeId }: Props) => {
     const program = useTrackerProgram(programId);
 
     // Being pragmatic here, disabling behavior we will implement later
-    const commonStateManagementProps: Object = useWorkingListsCommonStateManagement(storeId, TEI_WORKING_LISTS_TYPE, program);
+    const {
+        orgUnitId,
+        lastTransaction,
+        lastTransactionOnListDataRefresh,
+        listDataRefreshTimestamp,
+        onAddTemplate,
+        onUpdateTemplate,
+        onDeleteTemplate,
+        onSetTemplateSharingSettings,
```

Comment:
One question, then plan is to have these variables here because we want to omit them from the `commonStateManagementProps`?

Reply:
Yeah, I don't particularly like this solution. At the moment, the variables shouldn't have been exposed from `useWorkingListsCommonStateManagement` at all (but rather moved to the event working list part). But all that aside, given we want to omit the variables, what would be your preferred solution? Should we make a custom utility function? Curious what you think.

Reply:
Will need to dive into it a little bit more. Give me sometime to think about it :)

Reply:
Usually my first thought would be to pass the props explicitly. Aka not having the `...passOnProps` but I understand that if there are 20 props in the `...passOnProps` this might be a little bit difficult.

Reply:
I agree, explicit should be the preferred option overall. But like you said, sometimes it is useful to get all except some of the props. I can think of a few solutions, but they're not as elegant.

Reply:
Yeah I cant think of a better one that what you did here 👍

Reply:
Ok, let's keep it for now then